doc/source/ray-observability/state/state-api.rst

```diff
@@ -0,0 +1,303 @@
+.. _state-api-overview-ref:
+
+Monitoring ray states
+=====================
+
+.. tip:: We'd love to hear your feedback on using Ray state APIs - `feedback form <https://forms.gle/gh77mwjEskjhN8G46>`_!
+
+Ray state APIs allow users to conveniently access the current state (snapshot) of Ray through CLI or Python SDK.
+
+.. tip:: APIs are pre-alpha and under active development. APIs are subject to change and not stable across versions.
+
+Getting Started
+---------------
+
+Run any workload. In this example, you will use the following script that runs 2 tasks and creates 2 actors.
+
+.. code-block:: python
+
+    import ray
+    import time
+
+    ray.init(num_cpus=4)
+
+    @ray.remote
+    def task_running_300_seconds():
+        print("Start!")
+        time.sleep(300)
+    
+    @ray.remote
+    class Actor:
+        def __init__(self):
+            print("Actor created")
+    
+    # Create 2 tasks
+    tasks = [task_running_300_seconds.remote() for _ in range(2)]
+
+    # Create 2 actors
+    actors = [Actor.remote() for _ in range(2)]
+
+    ray.get(tasks)
+
+Now, let's see the summarized states of tasks. It takes some time to query all data, so if it doesn't return the output immediately, retry the command.
+
+.. code-block:: bash
+
+    ray summary tasks
+
+.. code-block:: text
+
+    ======== Tasks Summary: 2022-07-22 08:54:38.332537 ========
+    Stats:
+    ------------------------------------
+    total_actor_scheduled: 2
+    total_actor_tasks: 0
+    total_tasks: 2
+
+
+    Table (group by func_name):
+    ------------------------------------
+        FUNC_OR_CLASS_NAME        STATE_COUNTS    TYPE
+    0   task_running_300_seconds  RUNNING: 2      NORMAL_TASK
+    1   Actor.__init__            FINISHED: 2     ACTOR_CREATION_TASK
+
+Let's list all running tasks.
+
+.. code-block:: bash
+
+    ray list tasks
+
+.. code-block:: text
+
+    ======== List: 2022-07-22 08:56:37.570942 ========
+    Stats:
+    ------------------------------
+    Total: 4
+
+    Table:
+    ------------------------------
+       FUNC_OR_CLASS_NAME        NAME                         SCHEDULING_STATE    TASK_ID                                           TYPE
```

Comment:
<img width="876" alt="image" src="https://user-images.githubusercontent.com/11676094/180498397-f32d4271-4958-4a54-9a72-2a27c496015e.png">

Not really relate to this PR, more of a cosmetic minor point. Should we change the default columns set for task to exclude "NAME" column? On a narrow terminal or window like this, it is not showing up too well.

Reply:
should we use `ray list actors` and `ray get actors xxx` for the list results example?